.github/workflows/ci.yml

```diff
@@ -1,9 +1,9 @@
 name: CI
 on: 
   push:
-  pull_request:
     branches:
-      - development
+      - main
+  pull_request:
 
```

Comment:
Just making sure, we're running these on every pull request, for merges sake, and any push to main correct?

Reply:
Yes, that's how I understand it too